jwst/tests_nightly/general/fgs/test_guider_pipeline.py

```diff
@@ -0,0 +1,89 @@
+import os
+import pytest
+from astropy.io import fits as pf
+from jwst.pipeline.calwebb_guider import GuiderPipeline
+
+
+pytestmark = [
+    pytest.mark.usefixtures('_jail'),
```

Comment:
Such usage of `_jail` can be good or bad. For instance, if you want to add a new test in this file that does not want that functionality, it would be impossible. It is also hard to read for someone that is not familiar with `pytest`. I understand that this is necessary now because of the requirement to not change ported tests too much. However, I recommend a rewrite going forward, so things are more explicit.

For example, in spacetelescope/hstcal#71, you can explicitly mark a base class however you like and then have tests inherit from the base class as needed. However, that is not the only solution available.

Reply:
I definitely agree with you. If/when tests are consolidated to single files and further broken down by test classes, I'd like to see something like `_jail` used as-needed, rather than forced.

Just something to keep in mind though... If you *don't* jail your test, or use `tmpdir` on your own, the directory where `pytest` was executed will become a minefield of input and output data. If those files are not in `.gitignore` there's a greater chance you will accidentally commit the data to the repository (this is especially true for anyone that relies on `git commit -a`).

Reply:
Yes, that is a risk. We can certainly defer this to a separate PR.